### DISCUSSION TITLE
Wiring pin out for rs485 Solis to USB

### DISCUSSION THREAD

**Discussion by MarkWeeman** [wills106/homeassistant-solax-modbus#732] [Q&A]
Hi there, I’ve scoured these forums to try and find a quick answer but not had any luck. 

Can anyone tell me which pin I should be using on the exceedconn rs485 connector to usb adapter. There like 4 ports to choose from

I’ve got the integrations set up, I’m just not pulling any data

**Comment by wills106** [comment]
You can find more details on the [Solis here](https://homeassistant-solax-modbus.readthedocs.io/en/latest/modbus-adaptor-setup/#solis-inverters) there's also a link to further information on GitHub
